### DISCUSSION TITLE
View event without editing

### DISCUSSION THREAD

**Discussion by tiagof** [saade/filament-fullcalendar#45] [Ideas]
Hi @saade 

At the moment - if I checked correctly - clicking an event does nothing if the user cannot edit that event, i.e. when `canEdit()` returns `false`.

I'd like to add a `canView()` method that (when defined, to prevent breaking changes) is used instead of `canEdit()` to decide whether to show the event modal.

Additionally, the event modal would only be editable if the `canEdit()` returns `true`.

Do you think this is a nice addition?
If so I'll prepare a PR....

**Comment by saade** [comment]
Hi @tiagof 

Yes! Feel free to open that PR :)

**Comment by tiagof** [comment]
@saade PR #46
